src/components/TopBar/TopBarMenu.vue

```diff
@@ -0,0 +1,396 @@
+<!--
+  - @copyright Copyright (c) 2022 Marco Ambrosini <marcoambrosini@icloud.com>
+  -
+  - @author Marco Ambrosini <marcoambrosini@icloud.com>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+	<NcActions v-if="!isSidebar"
+		v-shortkey.once="disableKeyboardShortcuts ? null : ['f']"
+		v-tooltip="t('spreed', 'Conversation actions')"
+		class="top-bar__button"
+		:aria-label="t('spreed', 'Conversation actions')"
+		:container="container"
+		@shortkey.native="toggleFullscreen">
+		<!-- White icon if in call -->
+		<template v-if="isInCall" #icon>
+			<DotsHorizontal :size="20"
```

Comment:
Should we should the settings wheel as before in case it's not a call?

15.0.0 | This PR
---|---
![Bildschirmfoto vom 2022-10-31 10-05-12](https://user-images.githubusercontent.com/213943/198971524-88e2e471-2afb-4eee-9b89-5cce4751480f.png) | ![Bildschirmfoto vom 2022-10-31 10-05-05](https://user-images.githubusercontent.com/213943/198971539-60247184-4439-4ef8-9246-573ce1acfb1b.png)

Reply:
I don't think so, as it contains (and will likely contain more) actions that are not settings